README.md

```diff
@@ -129,11 +131,68 @@ add_subdirectory(Face_Detector.Cpp)
 target_link_libraries(YOUR_EXECUTABLE CLFML::face_detector)
 ```
 
+## Building a ROS2 package with Colcon
+Before using this library you will need the following packages installed:
+- OpenCV
+- ROS2
+- ROS CV bridge
+- Working C++ compiler (GCC, Clang, MSVC (2017 or Higher))
+- CMake
+
+### Running the examples (Ubuntu, CPU)
+
+1. Clone this repo:
+```
+git clone https://github.com/CLFML/Face_Detector.Cpp.git
+```
+
+2. Make sure to set the ROS2 switch in CMakeLists.txt:
+
+```cmake
+option(CLFML_ROS2_PACKAGE_BUILD "Build a ROS2 package" ON)
+```
```

Comment:
This is normally not the way to enable a cmake option;

Normally you would pass it in to the command for generating:

```bash
cmake . -B build -G Ninja -DCLFML_ROS2_PACKAGE_BUILD =ON
```
or setting it on the VSCode side in launch.json:

```json
"cmake.configureSettings": {
    "CLFML_ROS2_PACKAGE_BUILD": "ON"
}
```
--cmake-args
Or you could do it in colcon using the --cmake-args options:

```bash
colcon build --package-select face_detector --cmake-args "-DCLFML_ROS2_PACKAGE_BUILD =ON"
```

Or you can create a .pkg file and specify it in here [see this example](https://github.com/eProsima/Micro-CDR/blob/1c712a08cf5571af0435bf1d881075e813c09513/colcon.pkg#L5)

Reply:
Fixed it now using a colcon.pkg file, so please ignore this comment

Reply:
But remove the text specifying that you need to change anything in cmake, now it does this all automatically using the colcon.pkg file.  Remove the line about overriding the cache too